flocker/node/agents/functional/test_cinder.py

```diff
@@ -396,7 +415,19 @@ def test_get_device_path_no_attached_disks(self):
 
         self.assertEqual(device_path.realpath(), new_device)
 
-    @require_virtio
+
+@require_virtio
```

Comment:
The tests that had this skip were soooooooooooooooo sloooooooooooooooow to discover they should skip it was absurd.  I couldn't stop myself from fixing them so they skip super fast instead.